packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -432,13 +434,13 @@ export class SmoothControls extends EventDispatcher {
     this.moveCamera();
   }
 
-  setTouchActionStyle() {
+  updateTouchActionStyle() {
     const {style} = this.element;
 
     if (this._interactionEnabled) {
       const {touchAction} = this._options;
       if (this._disableZoom && touchAction !== 'none') {
-        style.touchAction = `${touchAction} pinch-zoom`;
+        style.touchAction = 'manipulation';
```

Comment:
Produces more consistent and intuitive behavior on Android.
Hoping it's the same for iOS.